agents/plugins/mk_oracle

```diff
@@ -1736,7 +1747,7 @@ sql_instance () {
                          || '|' || round(nvl(popen_time, -1))
                          || '|' || pblock_size
                   from(
-                      select i.instance_name, i.version, i.status, i.logins, i.archiver
+                      select i.instance_name, i.${version_column} version, i.status, i.logins, i.archiver
```

Comment:
select i.instance_name, i.${version_column}, i.status, i.logins, i.archiver

Reply:
Removing the version from 'i.${version_column} version' will result in a syntax error in SQL, because the column name in the subquery will change when ${version_column} changes the value. I tried it without version during testing and it won't work.

**Adding the version alias in the subquery is mandatoiry!**

Reply:
@lasoe So would it be OK for you to keep `i.${version_column} version`?

Reply:
@jherbel Yes this is OK for me. I hadn't considered the impact on the subquery.